cli/configssh.go

```diff
@@ -501,6 +508,14 @@ func (r *RootCmd) configSSH() *clibase.Cmd {
 			Description: "Specifies the path to an SSH config.",
 			Value:       clibase.StringOf(&sshConfigFile),
 		},
+		{
+			Flag:    "coder-cli-path",
```

Comment:
How about just using `binary-path` (without prefix)? I feel like `cli` terminology isn't something we expose the users to in association with the `coder` command, and may cause a disconnect.

Reply:
One thing I was trying to make clear in the flag was this was the `coder` binary path, not anything else like the `ssh` binary. 

Is `coder-binary-path` too long?

Reply:
Nah, not too long IMO, your reasoning makes sense 👍🏻